components/cam/include/CamFile.h

```diff
@@ -0,0 +1,53 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2021 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <array>
+#include <string>
+#include <vector>
+
+#include <glm/glm.hpp>
+
+namespace openblack::cam
+{
+
+struct CamHeader
+{
+	uint32_t unk1;           // No perceptible difference when altered. Some values cause the game to crash
+	uint32_t movementSpeed;  // The larger this integer, the slower the movement speed
+	uint32_t numberOfPoints; // How many path vertices there are in the file
+};
+
+struct CamPoint
+{
+	glm::vec3 position;
+	glm::vec3 lookAtPosition;
```

Comment:
Since `CameraPath::LoadFromFile` copies this data over to store in a `CameraPoint`, you could use `std::array<float, 3>` which would allow us to have this component exist without an explicit 3rd party dependency like glm.

Instead of the copy, you would use `glm::make_vec3(point.position.data())`